frontend/src/registration/components/CongratulationsPage/CongratulationsPage.tsx

```diff
@@ -45,6 +47,11 @@ export class CongratulationsPage extends React.PureComponent<Props, State> {
           <p>
             <WrappedMessage messages={messages} id="congratsMessage2" />
           </p>
+          <NavLink exact to={ROUTES.Console.HOME}>
+            <Button>
```

Comment:
Can you use the larger button, and center it in the view? 

```suggestion
            <Button size="lg">
```

Reply:
To center the component, you can wrap a div around it with `className="text-center"`.

Reply:
Thanks for it, I have added a container which centers the button and used large class for enlarging the button